src/lib/storage/value_segment.hpp

```diff
@@ -25,10 +25,10 @@ class ValueSegment : public BaseSegment {
   // Return all values. This is the preferred method to check a value at a certain index. Usually you need to
   // access more than a single value anyway.
   // e.g. const auto& values = value_segment.values(); and then: values[i]; in your loop.
-  const std::vector<T>& values() const;
+  const std::vector<T>& values() const { return _values; };
 
  protected:
-  // Implementation goes here
+  std::vector<T> _values;
```

Comment:
The way I understand
> Private / proctected members / methods: like variables / functions with leading underscore, e.g., `_get_chunks()`

the underscore should be removed.

```suggestion
  std::vector<T> values;
```

Reply:
Since the slashes in your quoted text indicate alternatives, I understand in regarding to that variable as: ~~Private~~ / protected members / ~~methods~~: like **variables** / ~~functions~~ with leading underscore.

Reply:
I think you a right.